task/conftest-clair/0.1/conftest-clair.yaml

```diff
@@ -33,8 +33,16 @@ spec:
                    timestamp: $date,
                    namespace,
                    successes,
-                   failures: (.failures // [])|map(.metadata.details.name) | unique
+                   failures: (.failures | length)
                  }' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json || true)
+
+        # Log out the failing runs
+        if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+        then
+          echo "The clair-vulnerabilities test fails with the following runs:"
+          echo $(jq '.[].failures' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json)
```

Comment:
Same as above, we can use the `.[].failures // []|map(.metadata.details.name)` jq filter here to display only the error names.